code/modules/reagents/reagents.dm

```diff
@@ -152,6 +154,9 @@
 
 /datum/reagent/proc/affect_ingest(var/mob/living/carbon/M, var/alien, var/effect_multiplier)
```

Comment:
change this line to:
`/datum/reagent/proc/affect_ingest(mob/living/carbon/M, alien, effect_multiplier)`

Along with all the procs for the files you are changing. Remove the `var/` used as arguments in the procs.

Reply:
oh ok. i didn't even tried to mess with that since it was already there and i didn't wanted to mess it up. will do